### PR TITLE
Gainstation

### PR DESCRIPTION
**Pull request by Crashtest-LXXXIX**
## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by TotallyNotADog** [comment]
from what i can tell, this is supposed to be a PR to make it so that you are happy from being fat instead of sad, but
1. it's implemented so that _everyone_ has it changed to be positive
2. for some reason it deletes a billion .dmis unrelated to it

tl;dr try again, put this stuff into a quirk rather than chaning it outright

**Comment by Dahlular** [comment]
can you uh, format your PR correctly

**Comment by ArchieBeepBoop** [comment]
Only way I see this being a-okay is if it becomes a quirk, yeah. Also, for balance reasons instead of +8, I'd use +4 and have it be a neutral trait that when the character is fat, it gives you the positive, but when they are not, it goes negative.

**Comment by ArchieBeepBoop** [comment]
Also, to update your branch instead of doing pull requests like that, add the upstream repository as a remote, fetch the upstream repo and then git merge upstream/master

**Comment by Crashtest-LXXXIX** [comment]
right had the pull request the wrong way around
